### PR TITLE
APEXMALHAR-2479-regexparser-example

### PR DESCRIPTION
**Pull request by venkateshkottapalli**
@shubham-pathak22 Please review the regex parser example.

### CODE REVIEW

File: examples/parser/src/main/java/org/apache/apex/examples/parser/regexparser/ServerLogGenerator.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  }
+
+  @Override
+  public void emitTuples()
+  {
+    while (tuplesEmmitedinWindow < tupleRate) {
+      String line = "2015-10-01T03:14:49.000-07:00 lvn-d1-dev DevServer[9876]: INFO: [EVENT][SEQ=248717]" +
+          " 2015:10:01:03:14:49 101 sign-in_id=11111@psop.com ip_address=1.1.1.1  service_id=IP1234-NPB12345_00 " +
+          "result=RESULT_SUCCESconsole_id=0000000138e91b4e58236bf32besdafasdfasdfasdfsadf  account_id=11111  platform=pik";
+      outputPort.emit(line.getBytes());
```
Comment (shubham-pathak22) [inline examples/parser/src/main/java/org/apache/apex/examples/parser/regexparser/ServerLogGenerator.java:48]: Could you add few invalid records as well to show that those will be emitted from error port?

**Comment by shubham-pathak22** [comment]
@venkateshkottapalli I have added my comments. Also make sure travis build check passes.

File: examples/parser/src/main/java/org/apache/apex/examples/parser/regexparser/ServerLogGenerator.java (hunk 1 above)
Comment (venkateshkottapalli) [reply examples/parser/src/main/java/org/apache/apex/examples/parser/regexparser/ServerLogGenerator.java:48]: @shubham-pathak22 added the error inputs as well.

**Comment by venkateshkottapalli** [comment]
@shubham-pathak22  Please check now.

File: examples/parser/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ This app showcases **Json Parser**. Data generator sends Json data to the Json P
 
 * **Csv Parser App**
 
-This application showcases how to use CsvParser from [Apex Malhar](https://github.com/apache/apex-malhar) library. The CsvParser converts your delimited data to a key-value pair map or concrete java class also know as [POJO](https://en.wikipedia.org/wiki/Plain_Old_Java_Object). The parser emits key-value pair map on *parsedOutput* port. It emits POJO on *out* and error records on *err* port.
+This application showcases how to use [CsvParser](https://datatorrent.com/docs/apidocs/com/datatorrent/contrib/parser/CsvParser.html) from [Apex Malhar](https://github.com/apache/apex-malhar) library. The CsvParser converts your delimited data to a key-value pair map or concrete java class also know as [POJO](https://en.wikipedia.org/wiki/Plain_Old_Java_Object). The parser emits key-value pair map on *parsedOutput* port. It emits POJO on *out* and error records on *err* port.
```
Comment (tweise) [inline examples/parser/README.md:8]: Why are these links pointing to the datatorrent web site and not to the ASF hosted Javadoc?

**Comment by tweise** [comment]
@shubham-pathak22 since you are responsible for the merge, I would expect follow-up from you on this.

See also https://issues.apache.org/jira/browse/APEXMALHAR-2477, perhaps there can be a broader sweep.

@amberarrow FYI

**Pull request merged**